.travis.yml

```diff
@@ -0,0 +1,16 @@
+sudo: required
+dist: trusty
+language: php
+php: 7.2
+env: WP_VERSION=latest WP_MULTISITE=0 WP_CORE_DIR=/tmp/wordpress
```

Comment:
Not sure if this matters in this instance, but the 5ftf site is on a multisite.

Reply:
Good point– I've changed that, and also added the multisite flag to `phpunit.xml.dist` so that local tests also run as multisite.